### PR TITLE
Use Accept-Language Header

### PR DESCRIPTION
**Pull request by kobergj**
Use the widely accepted `Accept-Language` Header instead of the custom `Prefered-Language` one

### CODE REVIEW

File: services/userlog/pkg/service/http.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // HeaderPreferedLanguage is the header where the client can set the locale
-var HeaderPreferedLanguage = "Prefered-Language"
+var HeaderPreferedLanguage = "Accept-Language"
```
Comment (kulmann) [inline services/userlog/pkg/service/http.go:11]: plus it makes a typo go away 😁

**Review by kulmann — APPROVED** [review]
thank you 🤩

File: services/userlog/pkg/service/http.go [hunk 2]
```diff
@@ -7,8 +7,8 @@ import (
 	revactx "github.com/cs3org/reva/v2/pkg/ctx"
 )
 
-// HeaderPreferedLanguage is the header where the client can set the locale
-var HeaderPreferedLanguage = "Prefered-Language"
+// HeaderPreferredLanguage is the header where the client can set the locale
+var HeaderPreferredLanguage = "Accept-Language"
```
Comment (butonic) [inline services/userlog/pkg/service/http.go:11]:
```suggestion
// HeaderAcceptLanguage is the header where the client can set the locale
var HeaderAcceptLanguage = "Accept-Language"
```

File: services/userlog/pkg/service/http.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ func (ul *UserlogService) HandleGetEvents(w http.ResponseWriter, r *http.Request
 		return
 	}
 
-	conv := NewConverter(r.Header.Get(HeaderPreferedLanguage), ul.gwClient, ul.cfg.MachineAuthAPIKey, ul.cfg.Service.Name, ul.cfg.TranslationPath, ul.registeredEvents)
+	conv := NewConverter(r.Header.Get(HeaderPreferredLanguage), ul.gwClient, ul.cfg.MachineAuthAPIKey, ul.cfg.Service.Name, ul.cfg.TranslationPath, ul.registeredEvents)
```
Comment (butonic) [inline services/userlog/pkg/service/http.go:34]:
```suggestion
	conv := NewConverter(r.Header.Get(HeaderAcceptLanguage), ul.gwClient, ul.cfg.MachineAuthAPIKey, ul.cfg.Service.Name, ul.cfg.TranslationPath, ul.registeredEvents)
```

Comment (kobergj) [reply services/userlog/pkg/service/http.go]: partypooper

**Pull request merged**